simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/evse/EvseState.java

```diff
@@ -0,0 +1,12 @@
+package com.evbox.everon.ocpp.simulator.station.evse;
+
+/**
+ * Represents a state of station EVSE.
+ *
+ * AVAILABLE corresponds to Operative
+ * UNAVAILABLE corresponds to Inoperative
+ */
+public enum EvseState {
```

Comment:
I'd rename this enum to something with word Availability as it's not really the whole state

Reply:
The spec says: _When changing the availability of an EVSE to Operative, the status of the EVSE has changed to Available, Occupied or Reserved._
So EVSE has a status in our terms its state. Perhaps we should rename it to EvseStatus and use Status in other places according to spec.

Reply:
Status is indeed better, State is confusing as the status is not whole Evse state

Reply:
👍 renamed